Source/WebCore/css/StyleRuleImport.cpp

```diff
@@ -135,7 +135,7 @@ void StyleRuleImport::requestStyleSheet()
 
     // FIXME: Skip Content Security Policy check when stylesheet is in a user agent shadow tree.
     // See <https://bugs.webkit.org/show_bug.cgi?id=146663>.
-    CachedResourceRequest request(absURL, CachedResourceLoader::defaultCachedResourceOptions(), std::nullopt, String(m_parentStyleSheet->charset()));
+    CachedResourceRequest request(absURL, CachedResourceLoader::defaultCachedResourceOptions(), std::nullopt, String::fromLatin1(m_parentStyleSheet->charset()));
```

Comment:
Why do we need `::fromLatin1()` here? There is a `String(ASCIILiteral)` constructor available, which would be more efficient.

Reply:
I used the `::fromLatin1()` because I saw it used elsewhere. I thought that since we know the `ASCIILiteral` is guaranteed to be 8-bit and Latin1, that it would be able to avoid some checks. But clearly not!